packages/nextjs-contentful-renderer/src/utils/isReplica/isReplica.ts

```diff
@@ -0,0 +1,5 @@
+import type { Replica } from '../contentful/getRouteBySlug/types';
+
+export const isReplica = (data): boolean => {
```

Comment:
we should implement this using [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) as this will allow the compiler to properly infer the type